Code/api/services/mailing.js

```diff
@@ -0,0 +1,129 @@
+
+var User = require('../models/users'),
+    Support = require('../models/support'),
+    nodemailer = require("nodemailer")
+
+
+var headers = {
+    accountPinRecovery: "0xAccountPinRecovery"
+},
+    pinRecoveryMailExpiry = 1000 * 3600 * 1,
+    SMTP_config = {
+        service: "Gmail",
+        auth: {
+            user: 'vvega019@fiu.edu',

```

Comment:
your email cannot be hardcoded here ....
use the vipmailer            auth: {
                    user: 'fiuvipmailer@gmail.com',
                    pass: 'vipadmin123'
                }

UserRotutes.js